deck/views.py

```diff
@@ -47,7 +51,7 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class CreateEvent(BaseEventView, CreateView, FormValidRedirectMixing):
+class CreateEvent(LoginRequiredMixing, BaseEventView, CreateView, FormValidRedirectMixing):
```

Comment:
I followed the pattern from FormValidRedirectMixing. Should I change both classes?